internal/idler/user_idler.go

```diff
@@ -35,6 +37,7 @@ type UserIdler struct {
 	features        toggles.Features
 }
 
+// NewUserIdler creates an instance of UserIdler
```

Comment:
You could be a bit more specific. You just state the obvious. I am not too fuzzed atm, but a good godocs would describe the parameters as well and give more info about the return type, for example that a pointer to a UserIdler is returned